src/seq/aminoacid.jl

```diff
@@ -308,11 +372,13 @@ copy(seq::AminoAcidSequence) = orphan!(AminoAcidSequence(seq.data, seq.part))
 
 start(seq::AminoAcidSequence) = seq.part.start
 
+
 function next(seq::AminoAcidSequence, i)
     aa = seq.data[i]
     return (aa, i + 1)
```

Comment:
Do we need this intermediate variable here? Maybe the compiler can optimize this, but still, I believe it's much clearer to don't use `aa` here (and maybe transforming this function definition in a one-liner).